src/new-client/src/App/Footer/Footer.tsx

```diff
@@ -1,10 +1,14 @@
 import ContactUsButton from "./ContactUsButton"
 import StatusBar from "./StatusBar"
 import { StatusButton } from "./StatusButton/StatusButton"
```

Comment:
Remember to use index files to export the correct contracts of each module.

I was expecting to find something like: `import { StatusButton, statusButton$} from './StatusButton'`